test/infamy/transport.py

```diff
@@ -1,6 +1,14 @@
 from abc import ABC, abstractmethod
 from infamy.neigh import ll6ping
 
+def infer_put_dict(name, models):
+    if not models.get("ietf-system"):
+        models["ietf-system"] = { "system": { "hostname": name} }
+    else:
+        if not models["ietf-system"].get("system"):
+            models["ieft-system"]["system"] = {}
+        if not models["ietf-system"]["system"].get("hostname"):
+            models["ietf-system"]["system"]["hostname"] = name
```

Comment:
```suggestion
    hostname = { "ietf-system": { "system": { "hostname": name } } }
    if not models.get("ietf-system", {}).get("system", {}).get("hostname"):
        models |= hostname
```

I think this suggestion is functionally equivalent, and maybe a bit more readable

Reply:
Very neat, but not possible to use | with nested dictionaries, unfortunately.